.github/workflows/linter.yml

```
@@ -13,7 +13,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
     - name: Install dependencies
-      run: pip install flake8
+      run: pip install flake8 flake8-docstrings flake8-rst-docstrings
```

```suggestion
      run: pip install -r requirements-ci.txt
```
Since these are already a dependency, this would be a more general solution which we do not have to update in case we add more plugins.

---

Sounds good.